src/selectors/selectors.ts

```diff
@@ -31,6 +35,18 @@ const getSelectedFeedPosts = (state: AppState, feedUrl: string) => {
         });
 };
 
+export const getContactFeeds = createSelector([ getContacts ], (contacts) => {
+    const mutualContacts = contacts.filter(contact => contact.type === 'mutual-contact') as MutualContact[];
```

Comment:
I've found a way to use type guards for arrays too:
```TypeScript
const mutualContacts = contacts.filter((contact): contact is MutualContact => contact.type === 'mutual-contact')
```
I believe this should work here
See https://github.com/microsoft/TypeScript/issues/7657